.travis.yml

```diff
@@ -12,7 +12,7 @@ matrix:
   allow_failures:
     - python: "nightly"
 install:
-  - pip install --upgrade coveralls pytest "pytest-cov>=2.5.1" dataclasses typing_extensions
+  - pip install -e '.[dev]'
   # mypy can't be installed on pypy
   - if [[ "${TRAVIS_PYTHON_VERSION}" != "pypy"* ]] ; then pip install mypy ; fi
   # Black is Python 3.6+-only
```

Comment:
Oh, one thing I forgot: since we're moving development dependencies to `requirements-dev.txt` we need to move `mypy` and `black` installation there as well.

Reply:
Yes, it is.